wpeverest/everest-forms
# Fix - Double render of overall feedbak in quiz reporting

### All Submissions:






### Changes proposed in this Pull Request:



Previously, after form submit overall feedback renders twice. This PR solve that issue.

### How to test the changes in this Pull Request:

1. Goto Everest Forms > Builder Settings > Survey , Polls and quiz > Enable quiz reporting > enable overall feedback.
2. Set overall feedback and save form.
3. Preview form and submit.
4. Check overall feedback is correct or not.

Dependecy pr [Pro PR](https://github.com/wpeverest/everest-forms-pro/pull/958)
### Types of changes:




### Other information:




### Changelog entry

> Fix - Double render of overall feedbak in quiz reporting
